src/components/cc-map/cc-map.js

```diff
@@ -313,10 +308,13 @@ export class CcMap extends withResizeObserver(LitElement) {
   }
 
   render () {
-
     const noHeatmapPoints = (!this.error && this.mode === 'heatmap' && this.heatmapPoints?.length === 0);
     const errorMode = this.loading ? 'loading' : 'info';
 
+    if (this._map != null) {
+      this._map.invalidateSize();
+    }
```

Comment:
question: I tried but I don't understand why it works compared to the `onResize () { this._map.invalidateSize(); }`